beacon-chain/node/config.go

```diff
@@ -2,16 +2,26 @@ package node
 
 import (
 	"fmt"
+	"math"
 
 	"github.com/OffchainLabs/prysm/v6/cmd"
 	"github.com/OffchainLabs/prysm/v6/cmd/beacon-chain/flags"
+	storageFlags "github.com/OffchainLabs/prysm/v6/cmd/beacon-chain/storage/flags"
+	backfill "github.com/OffchainLabs/prysm/v6/cmd/beacon-chain/sync/backfill/flags"
+	"github.com/OffchainLabs/prysm/v6/config/features"
 	"github.com/OffchainLabs/prysm/v6/config/params"
 	"github.com/OffchainLabs/prysm/v6/consensus-types/primitives"
 	"github.com/OffchainLabs/prysm/v6/monitoring/tracing"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/urfave/cli/v2"
 )
 
+const (
+	maxBlobRetentionEpoch         = math.MaxUint32
```

Comment:
Isn't this defined somewhere already?

Reply:
Not exactly, there is no defined max retention epoch so we just use a number that is big enough here